ios/Classes/MapboxMapController.swift

```diff
@@ -525,6 +525,15 @@ class MapboxMapController: NSObject, FlutterPlatformView, MGLMapViewDelegate, Ma
             mapView.style?.addSource(source)
 
             result(nil)
+
+        case "style#sourceExists":
+            guard let arguments = methodCall.arguments as? [String: Any] else { return }
+            guard let sourceId = arguments["sourceId"] as? String else { return }
+            guard let source = mapView.style?.source(withIdentifier: sourceId)
```

Comment:
convert to an if? source it not used so a guard statement might not be the best solution.